Src/Web/Web.Shared.Net/ApplicationInsightsHttpModule.cs

```diff
@@ -18,6 +18,9 @@ public sealed class ApplicationInsightsHttpModule : IHttpModule
     {
         private readonly RequestTrackingTelemetryModule requestModule;
         private readonly ExceptionTrackingTelemetryModule exceptionModule;
+        private MethodInfo addOnSendingHeadersMethod;
+        private bool? addOnSendingHeadersMethodExists;
+        private object[] paramsForReflectiveCall;
```

Comment:
Name should be reflective of content. This works better: `addOnSendingHeadersMethodParams`